sunscreen_compiler_common/src/transforms/graph_transforms.rs

```diff
@@ -161,8 +168,14 @@ where
                 }
             };
 
+            if let Some(node) = inserted_node {
```

Comment:
Now that I think about it, the caller can just iterate through `inserted_node_ids` and look for `Some(_)` entries.